frontend/app/src/next/components/errors/unauthorized.tsx

```diff
@@ -21,29 +19,3 @@ export function Unauthorized() {
     </div>
   );
 }
-
-export function WrongTenant({ desiredTenantId }: { desiredTenantId: string }) {
```

Comment:
just to sanity check here the idea is we never have a url<>uuid pair that is incompatible

Reply:
yeah exactly, we can just never get into a state where we'd show this now. Unauthorized is possible still, but this isn't